app/views/shared/_error_messages.html.erb

```diff
@@ -2,6 +2,9 @@
   <% content_for :flash do %>
     <%= render ToastMessageComponent.new(icon: "x_circle", color: :red, messages_tag: :ul) do |c| %>
       <%= c.title do %>
+        <%# i18n-tasks-use t('activerecord.models.business') %>
+        <%# i18n-tasks-use t('activerecord.models.developer') %>
+        <%# i18n-tasks-use t('activerecord.models.user') %>
```

Comment:
Is this saying the entire scope is used? I think it should be a bit more specific, like `activerecord.models.business.errors.messages.not_saved`, right? Or maybe I'm misunderstanding how this is working.

Reply:
It's being used in the latter part of the line.

```rb
 <%= I18n.t "errors.messages.not_saved", count: resource.errors.count, resource: resource.class.model_name.human.downcase %>
```

specifically:

```rb
resource.class.model_name.human.downcase
```
[According to Rails Guides (i18n):](https://guides.rubyonrails.org/i18n.html#active-model-methods)

> `model_name.human` and `human_attribute_name` use translations for model names and attribute names if available in the activerecord.models scope


Usage:

```rb
pry(main)> I18n.with_locale(:es) { Developer.model_name.human }
=> "Desarrollador"
pry(main)> I18n.with_locale(:es) { I18n.t "errors.messages.not_saved", count: 2, resource: Developer.model_name.human }
=> "Ocurrieron 2 errores al tratar de guardar Desarrollador:"
```

Reply:
But is your comment saying that the _entirety_ of that resource is being used? Or only the "not saved" error message?

Reply:
Ah, I see. 

More specifically, it's using `activerecord.models.(model).one`

Reply:
Got it, thanks for clarifying!